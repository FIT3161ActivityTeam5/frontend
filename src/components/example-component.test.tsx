import React from 'react';
import renderer from 'react-test-renderer';

import ExampleComponent from './example-component';

describe('<ExampleComponent />', () => {
  // Snapshot test.
  it('renders correctly', () => {
    const tree = renderer.create(<ExampleComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
