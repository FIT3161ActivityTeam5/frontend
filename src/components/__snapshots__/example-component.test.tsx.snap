// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExampleComponent /> renders correctly 1`] = `
<Text>
  Hello world!
</Text>
`;
